do-catalog/src/views/DatasetData.vue

```diff
@@ -76,9 +90,14 @@
 
 <script>
 import { mapState } from 'vuex';
+import Button from '@/components/Button.vue';
+import { formUrl } from '@/utils/form-url';
```

Comment:
I'm not sure about spliting this small function into another file.

Reply:
As talked before, the formUrl function will be needed in other areas of the page, so is better to keep it separated